src/components/aboutUsCard/index.css

```diff
@@ -0,0 +1,10 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.cardTitle {
+  color: #590995;
+  margin: 1.8em 0;
+}
+.cardPar {
```

Comment:
What is cardPar, please don’t abbreviate and use meaningful names

Reply:
I replaced it with cardParagraph